api/v2.0/swagger.yaml

```diff
@@ -2230,6 +2230,131 @@ paths:
           $ref: '#/responses/403'
         '500':
           $ref: '#/responses/500'
+  /usergroups:
+    get:
+      summary: Get all user groups information
+      description: Get all user groups information
+      tags:
+        - usergroup
```

Comment:
use `operationId: listUserGoups` to this API

Reply:
include the `requestId` in the parameters?

Reply:
Updated